src/stores/ConfigStore.ts

```diff
@@ -20,6 +21,7 @@ type CleengData = {
 type ConfigState = {
   config: Config;
   accessModel: AccessModel;
+  environments: { host: string; player: string };
```

Comment:
I don't know if I'm comfortable having these values in the Config, since loading the config depends on the host in the first place. Maybe they can be added to the 'settings' instead?